lyrebird/src/library.rs

```diff
@@ -229,14 +229,12 @@ impl MusicLibrary
 					}
 				}
 			)
-			.and_then
+			.map
 			(
 				|files|
 				{
 					let files = BTreeSet::from_iter(files.iter());
-					Some
-					(
-						files
+					files
```

Comment:
Given this Some() removal, the rest of the adjoined lines should be de-indented too so we maintain one level of tab indent here for the trailing calls off `files` please.